departments.ps1

```diff
@@ -244,7 +244,7 @@ Returns the raw JSON data containing all workers from ADP Workforce
 
             do {
                 $result = $null
-                $urlOffset = $Url + "?$" + "skip=$skip&$" + "top=$take"
+                $urlOffset = $Url + "?$" + "skip=$skip&$" + "top=$take&$" + "filter=MUT_INK_SOORT ne 31 and MUT_INK_SOORT ne 63 and MUT_INK_SOORT ne 62 and MUT_INK_SOORT ne 53 and /workers/workerDates/terminationDate gt $($filterDate)"
```

Comment:
The variable $filterDate is not configured in the script. Is this something that should be configured in the configuration.json?

I have expirense with this source connector. Is $filterDate something that should be added to the person script?


Another question, is the line below something that applies to all customers or only for one?

"filter=MUT_INK_SOORT ne 31 and MUT_INK_SOORT ne 63 and MUT_INK_SOORT ne 62 and MUT_INK_SOORT ne 53"

Reply:
This has been corrected